content/en/getting_started/synthetics/api_test.md

```diff
@@ -64,7 +70,9 @@ For more information on how to set up private locations, see [Getting Started wi
 
 ### Specify test frequency
 
-Select the frequency of your test runs.
+Select the frequency of your test runs. Additionally, on-demand and continuous integration pipelines tests are available. 
+
+For more information, see [Synthetic CI/CD Testing][6].
```

Comment:
```suggestion
```

Reply:
Hey @MargotLepizzera, I thought we agreed we would add this link to both API test and Browser test Getting Started pages. Would you like me to delete this link from the other page as well?